README.md

```diff
@@ -58,7 +58,11 @@ Below are the properties allowed by the Changelog Enforcer. These properties are
 
 #### `versionPattern`
 * Default: `'## \\[((v|V)?\\d*\\.\\d*\\.\\d*-?\\w*|unreleased|Unreleased|UNRELEASED)\\]'`
-* A regex pattern used to extract the versions from the changelog. Changelog Enforcer assumes the use of the KeepAChangelog.com convention, and as such looks for a line starting with `## [version] - date`. Your regex should match the version as the 2nd match group. The regex pattern is used with global and multiline flags. Also note that since this is passed as a String, you will need to escape a backslash `\` character via `\\`
+* A regex pattern used to extract the versions from the changelog. Changelog Enforcer assumes the use of the [KeepAChangelog.com](https:// keepachangelog.com/en/1.0.0/) convention, and as such looks for a line starting with '## [version] - date. Versions are only extracted from the changelog when enforcing that the expected latest version (via the `expectedLatestVersion` property). 
+      
+  If you supply your own regex to match a different format, your regex must match the version as a capture group. The first capture group will be used if your regex includes mutliple. The regex pattern is used with global and multiline flags to find all of the versions in the changelog. 
+      
+  Because the regex is passed as a String, you will need to escape a backslash `\` character via `\\`
```

Comment:
Some suggestions to this entry:

```suggestion
* A regex pattern used to extract the version section headings from the changelog. Changelog Enforcer assumes the use of the [KeepAChangelog.com](https://keepachangelog.com/en/1.0.0/) convention for section headings, and as such looks for a line starting with `## [version] - date`. Versions are only extracted from the changelog when enforcing the expected latest version (via the `expectedLatestVersion` property).

  If you supply your own regex to match a different format, your regex must match the version as a capture group. The first capture group will be used if your regex includes multiple groups. The regex pattern is used with global and multiline flags to find all of the versions in the changelog. 

  Because the regex is passed as a String, you will need to escape backslash characters (`\`) character via `\\`.
```

Reply:
I'm also not sure about the wording "the KeepAChangelog.com convention", since the very first entry in their [FAQ](https://keepachangelog.com/en/1.0.0/#frequently-asked-questions) says that they don't prescribe any specific format as standard. Perhaps "the KeepAChangelog.com recommendation" would be more accurate?

Reply:
🤔 I see what you mean. However, their own wording also uses the term convention

> Not really. There's the GNU changelog style guide, or the two-paragraph-long GNU NEWS file "guideline". Both are inadequate or insufficient.
> This project aims to be a better changelog **convention**. It comes from observing good practices in the open source community and gathering them.
> Healthy criticism, discussion and suggestions for improvements are welcome.